Automattic/wp-calypso
# Interim update to ecommerce trial features

Related to:
* #74099

## Proposed Changes

* This PR implements two interim changes to the features list for free eCommerce trials:
  - It replaces the current Google Analytics card with a Payment setup card
  - It removes the CTA links for each of the cards, as the links weren't implemented
    * We are working on the CTAs and tracking for these items in #74099

### Screenshot

<img width="1241" alt="Free trial features" src="https://user-images.githubusercontent.com/3376401/223222002-711a3f1e-1d63-4848-a42a-f85fcb820b22.png">


## Testing Instructions



* Run this branch locally for via Calypso.live
* For a site on the free eCommerce trial, navigate to _Upgrades_ -> _Plans_ and then click on the "Free trials" tab
* Verify that none of the cards includes a CTA link
* Verify that you don't see a _Google Analytics_ card
* Verify that you see a _Get ready to be paid_ card -- it should appear in the second row

## Pre-merge Checklist



- [N/A] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)? _We want to get the change out ASAP, so we are not waiting for new strings_.
- [N/A] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?
